src/document.rs

```diff
@@ -147,12 +150,16 @@ fn value_to_string(value: &Value) -> String {
 ///             {"unsigned": 1000, "signed": -5, "float": 0.4},
 ///             schema,
 ///         )
-#[pyclass]
-#[derive(Default)]
+#[pyclass(module = "tantivy")]
```

Comment:
The module specification is necessary because of the usage of multiple impl blocks, right?